docs/Encoder.md

```diff
@@ -1,57 +1,57 @@
 
-## PCB Layout
+| Parameter                             | MIN   | NOM | MAX   |
+|---------------------------------------|-------|-----|-------|
+| Supply voltage Vcc                    | 4.5 V | 5 V | 5.5 V |
+| High-level input voltage (logic HIGH) | 2 V   |     | Vcc   |
+| Low-level input voltage (logic LOW)   | 0 V   |     | 0.8 V |
+| Common-mode input voltage             | -7 V  |     | 7 V   |
 
-All chips and d-sub connectors are located on the top layer. Signal routes from d-sub connectors to differential to single-ended converter and then to level shifter are all located on the top layer as well, while signals from level shifter to PicoZed are located in the intermediate level and each inverting and non-inverting pair of encoder signals are connected between each other by the termination resistor on the bottom layer. All signal traces have 6 mil thickness. Supply and ground traces connected to the chips have thickness of 10 mil.
+When the chip is operated at nominal conditions (Vcc = 5 V, room temperature 25 C), the output signals of the chip are such that the high-level output voltage is 3.8 V or more and low-level output voltage is around 0.2 V. Typical current the chip consumes is about 10 mA. This translates to about 50 mW of power consumption with 5 V supply. The rise and fall times of the output signal are both approximately equal to 4 ns. Assuming the system as first order, the bandwidth of the chip is approximately equal to 0.35/4 ns = 87.5 MHz.
 
-It is recommended that bypass capacitors are used on the power supplies to reduce noise, and, if possible, short trace lengths should be used to avoid excessive loading.
+Before connecting the encoder to AMDC, the user should check if the encoder output signals satisfy recommended operating conditions of the chip. More detailed information on the operating conditions can be found in the [datasheet](http://www.ti.com/lit/ds/symlink/am26c32.pdf).
 
 
+### 4. Level Shifter
+
+After the encoder differential signals are converted to single-ended, these signals are then level-shifted/translated to a lower voltage, allowing compatibility with PicoZed FPGA. The level shifter [SN74LVC8T245](http://www.ti.com/lit/ds/symlink/sn74lvc8t245.pdf) uses two separate configurable power-supply rails and allows for bidirectional operation. Signals on each side (input or output) are referenced to their respective voltage rail. In the AMDC REV D design, the input voltage rail is at 5 V and the output voltage rail is at 1.8 V. The maximum output current can be up to 32 mA when device is powered by 5 V. The outputs of this chip (6 signals in total if 2 encoders are connected) are then transmitted to PicoZed. The rise and fall times of the output signal are both approximately equal to 1ns. Assuming the system as first order, the bandwidth of the chip is approximately equal to 0.35/1 ns = 350 MHz.

```

Comment:
> The maximum output current can be up to 32 mA when device is powered by 5 V.

I am confused by this sentence. Is it saying that this part can source 32 mA to the picozed? And isn't the output voltage rail at 1.8 V (not 5 V)? Finally, if I understand all of that correctly, is this information relevant to the document or should we consider not providing it?